src/main/java/bio/terra/cli/context/WorkspaceUser.java

```diff
@@ -0,0 +1,108 @@
+package bio.terra.cli.context;
+
+import bio.terra.cli.command.exception.UserActionableException;
+import bio.terra.cli.context.utils.Printer;
+import bio.terra.cli.service.WorkspaceManagerService;
+import bio.terra.workspace.model.IamRole;
+import bio.terra.workspace.model.RoleBindingList;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * POJO class that represents a workspace user (i.e. someone a workspace is shared with). This is
+ * different from a regular {@link TerraUser} because they are never logged in. This is just a
+ * reference to another Terra user who has some level of workspace access. This class is not
+ * serialized to disk as part of the global context. It is used as the user-facing JSON output for
+ * commands that return a workspace user.
```

Comment:
Having a distinct layers for serialization would help to disambiguate Java business objects from disk serialization and command output.